nodejs/eks/cluster.ts

```diff
@@ -562,6 +563,20 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
         });
     }
 
+    // Setup OIDC provider to leverage IAM roles for k8s service accounts.
+    let oidcProvider = undefined;
+    if (args.createOidcProvider) {
+        const eksOidcProviderUrl = pulumi.interpolate `oidc.eks.${aws.getRegion().name}.amazonaws.com`;
+        const fingerprint = getIssuerCAThumbprint(eksOidcProviderUrl, OIDC_MAX_RETRIES, OIDC_SLEEP_INTERVAL); // Amazon root CA thumbprint
+        if (fingerprint) {
```

Comment:
Do we need to handle the else condition? Seems like it would be better to fail with an error if we are unable to create the OIDC provider as requested.